backend/src/libs/plugins/section-editing-permission/libs/constants/section-editing-route.constants.ts

```diff
@@ -0,0 +1,9 @@
+import { ApiPath } from '~/libs/enums/enums.js';
+import { SectionsApiPath } from '~/packages/sections/sections.js';
+
+const SECTION_EDITING_ROUTE = {
```

Comment:
it seems like an enum

Reply:
Enum of what? I thought it was wrong to declare it like enum, since it's two independent entities (path/method).

Reply:
enum = set of constants

```ts
const SECTION_EDITING_ROUTE_PATH = `/api/v1${ApiPath.SECTIONS}${SectionsApiPath.$ID}`;
const SECTION_EDITING_ROUTE_METHOD: HttpMethod = 'PUT';
```

should be

```ts
const SectionEditionRoute = {
  PATH: `/api/v1${ApiPath.SECTIONS}${SectionsApiPath.$ID}`,
  METHOD: 'PUT'
}
```

now your structure looks like just an object. if there is a chance to use something more familiar to other developers (ex. enum ), use it

Reply:
I define enum like set of related/connected constants (ex. ApiPath, CarBrand). Here I decided to group these 2 constants to object as they are logically associated. Probably would be better just split object into 2 constants.

Reply:
> Probably would be better just split object into 2 constants

but then it will be 2 constants with the similar context
and this is a case for merge them into 1 enum

Reply:
All right, I concede